packages/playground/src/weblets/tf_deployment_list.vue

```diff
@@ -484,29 +495,39 @@ watch(activeTab, () => (selectedItems.value = []));
 async function onDelete(k8s = false) {
   deletingDialog.value = false;
   deleting.value = true;
+
   try {
-    for (const item of selectedItems.value) {
-      try {
-        if (props.projectName?.toLowerCase() === ProjectName.Domains.toLowerCase()) {
-          await deleteGatewayDeployment(
-            updateGrid(grid, { projectName: props.projectName.toLocaleLowerCase() }),
-            item[0].workloads[0].name as string,
-          );
-        } else {
-          await deleteDeployment(updateGrid(grid!, { projectName: item.projectName }), {
-            deploymentName: item.deploymentName,
-            name: k8s ? item.deploymentName : item.name,
-            projectName: item.projectName,
-            ip: item.interfaces?.[0]?.ip,
-            k8s,
-          });
-        }
-      } catch (e: any) {
-        createCustomToast(`Failed to delete deployment with name: ${item.name}`, ToastType.danger);
-        console.log("Error while deleting deployment", e.message);
-        continue;
+    const projectNameLower = props.projectName?.toLowerCase();
+    const allSelectedItems = [...selectedItems.value];
+    selectedItems.value.forEach(item => {
+      if (item.projectName.toLowerCase().includes(ProjectName.Caprover.toLowerCase()) && item.workers) {
+        allSelectedItems.push(...item.workers);
       }
-    }
+    });
+
+    await Promise.all(
+      allSelectedItems.map(async item => {
+        try {
+          if (projectNameLower === ProjectName.Domains.toLowerCase()) {
+            await deleteGatewayDeployment(
+              updateGrid(grid, { projectName: projectNameLower }),
+              item[0].workloads[0].name as string,
+            );
+          } else {
+            await deleteDeployment(updateGrid(grid!, { projectName: item.projectName }), {
+              deploymentName: item.deploymentName,
+              name: k8s ? item.deploymentName : item.name,
+              projectName: item.projectName,
+              ip: item.interfaces?.[0]?.ip,
+              k8s,
+            });
+          }
+        } catch (e: any) {
+          createCustomToast(`Failed to delete deployment with name: ${item.name}`, ToastType.danger);
+          console.log("Error while deleting deployment", e.message);
```

Comment:
please use console.error to make it clear in the logger
```suggestion
          console.error("Error while deleting deployment", e.message);
```